README.md

```diff
@@ -114,26 +114,47 @@ RInAppMessaging.logEvent(CustomEvent(withName: "any_event_name_here", withCustom
 A preference is what will allow IAM to identify users for targeting and segmentation. At the moment, IAM will take in any of the following identifiers:
 
 1.  RakutenID - Any value that is considered by the app as user identifier.
-2.  UserID - The ID when registering a Rakuten account. e.g. an email address
-2.  IDTrackingIdentifier - The value provided by the internal identity SDK as the "tracking identifier" value.
-3.  AccessToken - This is the token provided by the internal RAuthentication SDK as the "accessToken" value
+1.  UserID - The ID when registering a Rakuten account. e.g. an email address
+1.  IDTrackingIdentifier - The value provided by the internal identity SDK as the "tracking identifier" value.
+1.  AccessToken - This is the token provided by the internal RAuthentication SDK as the "accessToken" value
 
 To help IAM identify users, please set a new preference every time a user changes their login state i.e. when they log in or log out.  
 After logout is complete please call  `registerPreference()` with nil parameter.  
-Not all identifiers have to be provided.  
-**NOTE:** For our internal users - to enable user targeting when you are using the internal RAuthentication SDK, you must provide an accessToken along with associated userId in `IAMPreferenceBuilder`.
+Not all identifiers have to be provided.
+
+#### Using the internal RAuthentication SDK
+To enable user targeting, you must provide an accessToken along with associated userId in `IAMPreferenceBuilder`.
 
 ```swift
 let preference = IAMPreferenceBuilder()
     .setUserId("testaccount@gmail.com")
-    .setRakutenId("testaccount")
     .setAccessToken("27364827346")
-    // .setIDTrackingIdentifier("998236")
     .build()
 
 RInAppMessaging.registerPreference(preference)
 ```
 
+#### Using the internal Identity SDK
+
+```swift
+let preference = IAMPreferenceBuilder()
+    .setIDTrackingIdentifier("998236")
+    .build()
+
+RInAppMessaging.registerPreference(preference)
+```
+
+#### Generic example using RakutenID
+
+```swift
+let preference = IAMPreferenceBuilder()
+    .setRakutenId("testaccount")
```

Comment:
I'd prefer "my-service-specific-id" instead of "testaccount" here.

And can we have a tech debt to rename this property to better reflect how it is intended to be used, maybe "ServiceSpecificId"?